tests/phpunit/tests/html/wpHtmlTagProcessorTest.php

```diff
@@ -1540,132 +1632,130 @@ public function test_next_tag_ignores_the_contents_of_a_script_tag( $script_then
 	}
 
 	/**
-	 * Data provider for test_ignores_contents_of_a_script_tag().
+	 * Data provider.
 	 *
-	 * @return array {
-	 *     @type array {
-	 *         @type string $script_then_div The HTML snippet containing script and div tags.
-	 *     }
-	 * }
```

Comment:
@hellofromtonya I was under the impression that @costdev used [this type annotation as a normative example](https://github.com/WordPress/wordpress-develop/pull/3920#discussion_r1089774919) - did I misunderstand?

Reply:
Core has a mixture of different documenting styles. This is not one that I've used, though I see it in Core. Looks like consensus is needed, then handbook and code updated for consistency (though out-of-scope for this PR).

Looks like I need to align with @costdev. We'll discuss to get alignment and then follow-up here.

Reply:
I'll let you make those reverts, then, unless you want me to. Happy to do that if you prefer.

Reply:
In talking with @costdev and others, for now, the updates in this PR will be used until there is consensus (via a yet to be opened Trac ticket) on the coding standard to adopt.

So the changes I pushed yesterday are the "for now" approach of simple(r) streamlined documentation and annotation.